### DISCUSSION TITLE
Can't highlight text outside of dialog when dialog is visible

### DISCUSSION THREAD

**Discussion by nerdyman** [ariakit/ariakit#805] [Questions] [answered]
Hi!

I've run into a slight issue with the `Dialog` component. I want to be able to click and highlight text outside of the dialog when it is open however It seems like the dialog is trapping focus. I'm using the same code used in the demo on the Reakit site so I'm unsure of why it's not working.

It works as expected on the Reakit website, however, when using the same code (without Reakit's CSS) in CodeSandbox I'm unable to select text.

This is the demo code from the Reakit site:

```js
import { Button } from "reakit/Button";
import { useDialogState, Dialog, DialogDisclosure } from "reakit/Dialog";

function Example() {
  const dialog = useDialogState();
  return (
    <>
      <p>Hello Joe!</p>
      <DialogDisclosure {...dialog}>Open dialog</DialogDisclosure>
      <Dialog {...dialog} tabIndex={0} aria-label="Welcome" hideOnClickOutside={false} modal={false}>
        <Button onClick={dialog.hide}>Close</Button>
      </Dialog>
    </>
  );
}

```

Reakit website:

![Peek 2020-12-03 10-35](https://user-images.githubusercontent.com/4908432/100998525-8b445100-3553-11eb-9424-ba5a9834eb2c.gif)

CodeSandbox https://rxk92.csb.app/ :

![Peek 2020-12-03 10-36-2](https://user-images.githubusercontent.com/4908432/100998506-85e70680-3553-11eb-9592-4d357bbd6818.gif)

Any help would be would be great 🙂

Also, thanks for all your work on Reakit - it's dope!

**Comment by nerdyman** [comment]
🤔 After stepping back through the versions it looks `reakit@1.1.1` works OK but anything higher traps focus https://jpgp2.csb.app/

**Comment by diegohaz** [comment, marked as answer]
Looks like it's a bug. Selecting text outside only works if there's a focusable container (`tabIndex={-1}`) wrapping the text (the website uses Gatsby, which adds a focusable wrapper to all pages). Could you please create an issue?

**Reply by nerdyman** [reply]
Thanks! That's fixed it.  I'll open an issue too. Edit: #806
